tower-layer/src/util/chain.rs

```diff
@@ -10,13 +10,7 @@ pub struct Chain<Inner, Outer> {
     outer: Outer,
 }
 
-/// Error's produced when chaining two layers together
-pub enum ChainError<I, O> {
-    /// Error produced from the inner layer call
-    Inner(I),
-    /// Error produced from the outer layer call
-    Outer(O),
-}
+type Error = Box<::std::error::Error + Send + Sync>;
```

Comment:
Teeniest of nits: I've been liking the clarity of `Box<dyn Trait>` (easier to differentiate from `Box<Struct>` (like if there were also `struct Error` somewhere)).

Reply:
fixed this one.